build.gradle

```diff
@@ -67,7 +67,7 @@ configurations {
 
 dependencies {
 
-    implementation ('org.eclipse.lsp4mp:org.eclipse.lsp4mp.ls:0.12.0-SNAPSHOT') {
+    implementation ('org.eclipse.lsp4mp:org.eclipse.lsp4mp.ls:0.12.0') {
```

Comment:
There is a second spot where LSP4MP needs to be updated: https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/build.gradle#L109

Reply:
Done